lms/djangoapps/course_api/blocks/transformers/proctored_exam.py

```diff
@@ -33,12 +33,9 @@ def collect(cls, block_structure):
         block_structure.request_xblock_fields('is_proctored_enabled')
         block_structure.request_xblock_fields('is_practice_exam')
 
-    def transform(self, usage_info, block_structure):
-        """
-        Mutates block_structure based on the given usage_info.
-        """
+    def transform_block_filter(self, usage_info, block_structure):
```

Comment:
Nit: make this method name plural since we are now returning a list: `transform_block_filters`.